docs/user/contextplugin.md

```diff
@@ -0,0 +1,19 @@
+# FlowForge Persistent Context Plugin
+
+Projects running within FlowForge include a plugin that make it possible
+to store values in persistent context in a containered environment.
+
+
+### Known Limitations
+
+Currently, only async versions of context `get` and context `set` functions are supported. 
+View the readme for more info.
```

Comment:
What readme? ./README.md doesn't tell me anything about this?

Reply:
There is a link below (within that section)

The read me has examples of what to do in the absence of the `sync` support.

![image](https://user-images.githubusercontent.com/44235289/208446806-113a4564-8de7-4bdd-9338-6094067e0adb.png)


How do you propose this should be written pls?

Reply:
outdated